index.html

```diff
@@ -1302,6 +1302,14 @@ <h2>Public Keys</h2>
 the <a>DID document</a>; see Section <a href="#authorization-and-delegation"></a>.
       </p>
 
+      <p class="note" title="Key Identifiers">
+The value of the <code>id</code> property may be structured as a <a href="https://en.wikipedia.org/wiki/Compound_key">compound key</a>. 
```

Comment:
On first read, this felt a bit strange to introduce the concept using database terminology. I'm wondering if there is an easier way to introduce this. Like, just tell people what the best practice is first and /then/ state that they can think of the example as analogous to compound database keys (although, I question the value of the explanation)... usually, unless the text adds something of great value, we can skip it.

Reply:
I'm expecting an objection from @selfissued regarding introducing more undefined terminology.

Reply:
Can I resolve this thread?